src/transformers/models/mobilebert/configuration_mobilebert.py

```diff
@@ -165,3 +168,14 @@ def __init__(
             self.true_hidden_size = hidden_size
 
         self.classifier_dropout = classifier_dropout
+
+
+class MobileBertOnnxConfig(OnnxConfig):
```

Comment:
Since MobileBERT has the same ONNX config as BERT, I suggest using the following:

```python
# Copied from transformers.models.bert.configuration_bert.BertOnnxConfig with Bert->MobileBert
class MobileBertOnnxConfig(OnnxConfig):
    @property
    def inputs(self) -> Mapping[str, Mapping[int, str]]:
        if self.task == "multiple-choice":
            dynamic_axis = {0: "batch", 1: "choice", 2: "sequence"}
        else:
            dynamic_axis = {0: "batch", 1: "sequence"}
        return OrderedDict(
            [
                ("input_ids", dynamic_axis),
                ("attention_mask", dynamic_axis),
                ("token_type_ids", dynamic_axis),
            ]
        )
```

This change will also add support for the `multiple-choice` head :)